feature/markdown_editor

HeorhiiPeshkov opened the pull request:
This commit introduces a new markdown editor feature, including:

- `markdown_editor/markdown_editor.py`: The main script for the markdown editor.
- `markdown_editor/output.md`: An initial output file for testing.
- Updated `.idea/workspace.xml` to reflect the new files and changes.

HeorhiiPeshkov committed:
feat: Add markdown editor functionality

This commit introduces a new markdown editor feature, including:

- `markdown_editor/markdown_editor.py`: The main script for the markdown editor.
- `markdown_editor/output.md`: An initial output file for testing.
- Updated `.idea/workspace.xml` to reflect the new files and changes.

OleksiiBreslavets commented on markdown_editor/markdown_editor.py:
```diff
@@ -0,0 +1,90 @@
+    return "\n".join(f"{i + 1}. {item}" for i, item in enumerate(items)) + "\n"
+
+def unordered_list(items):
+    return "\n".join(f"* {item}" for item in items) + "\n"
```
між функціями 2 рядки

HeorhiiPeshkov committed:
feat: Добавлен базовый функционал Markdown-редактора

Изменения:
- Обновлен файл workspace.xml для учета новых файлов и настроек.
- Добавлены функции форматирования текста в markdown_editor.py.
- Обновлен файл output.md с примером вывода Markdown.

OleksiiBreslavets commented on markdown_editor/markdown_editor.py:
```diff
@@ -0,0 +1,91 @@
+                continue
+        elif chafo == 'unordered-list':
+            num_items = int(input('Number of rows: '))
+            if num_items > 0:
```
while True:
                try:
 break
                except ValueError:

HeorhiiPeshkov committed:
Refreshed
